modules/security/build.gradle

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    alias(libs.plugins.shadow)
+}
+
+apply from: "$rootDir/buildscripts/java-core.gradle"
+apply from: "$rootDir/buildscripts/publishing.gradle"
+apply from: "$rootDir/buildscripts/java-junit5.gradle"
+apply from: "$rootDir/buildscripts/java-test-fixtures.gradle"
+
+description = 'ignite-security'
+
+dependencies {
+    implementation project(path: ':ignite-core')
+    annotationProcessor project(':ignite-configuration-annotation-processor')
+
+    implementation project(':ignite-api')
+    implementation project(':ignite-core')
+    implementation project(':ignite-configuration')
+    implementation libs.jetbrains.annotations
+    implementation libs.auto.service.annotations
+
+    annotationProcessor libs.auto.service
+
+    testImplementation project(':ignite-configuration')
+    testImplementation(testFixtures(project(':ignite-configuration')))
+    testImplementation libs.mockito.core
+    testImplementation libs.hamcrest.core
+    testImplementation libs.hamcrest.optional
+
+    testFixturesImplementation project(':ignite-configuration')
+    testFixturesImplementation libs.jetbrains.annotations
+
+}
+
+compileJava {
+    options.fork = true
+    options.forkOptions.jvmArgs += '-Dmicronaut.openapi.config.file=openapi/openapi.properties'
+}
+
+shadowJar {
+    archiveAppendix.set('shaded')
+    archiveClassifier.set('shaded')
+
+    minimize()
+
+    relocate('reactor', 'reactor.shaded')
```

Comment:
So we still shadow reactor-core here? Does this apply to all reactor-core dependencies we use (including the one brought by scalecube)?